### DISCUSSION TITLE
How to add a transparency effect to the badges?

### DISCUSSION THREAD

**Discussion by Zainab-Fahim** [coreui/coreui-react#294] [Q&A] [answered]
I intend to use [this](https://user-images.githubusercontent.com/80398726/174543972-3b31d7c9-427f-46a9-afc3-0ea484879cc9.png) effect on the badges.
<img width="326" alt="image1" src="https://user-images.githubusercontent.com/80398726/174543972-3b31d7c9-427f-46a9-afc3-0ea484879cc9.png">

I tried changing the value of `$success-base` ( \coreui-free-react-admin-template\src\scss\_variables.scss\89) to an alpha value, but that didn't work either.
Would be great if I could get an insight on how to solve this issue?
Thank You

**Comment by mrholek** [comment, marked as answer]
You can try to add a utility class - https://coreui.io/docs/utilities/opacity/
